manifests/config.pp

```diff
@@ -201,10 +201,11 @@
   }
 
   file { $log_dir:
-    ensure => directory,
-    mode   => $log_mode,
-    owner  => $log_user,
-    group  => $log_group,
+    ensure  => directory,
+    mode    => $log_mode,
+    owner   => $log_user,
+    group   => $log_group,
+    replace => $manage_log_dir,
```

Comment:
Should probably be:

```suggestion
    replace => $nginx::manage_log_dir,
```

… that or assign to a local variable like done already too many times above?  :clown_face:  

This module is not the finest in term of elegance :shrug:

Reply:
:man_facepalming:, fixed